Sources/SwifterAuth.swift

```diff
@@ -157,24 +157,21 @@ public extension Swifter {
     }
     
     #endif
-    
+
     @discardableResult
-    class func handleOpenUrlSSO(_ url: URL, callbackUrl: URL) -> Bool {
-        guard url.hasSameUrlScheme(as: callbackUrl) else {
+    class func handleOpenURL(_ url: URL, callbackURL: URL, isSSO: Bool = false) -> Bool {
```

Comment:
The function signature changed; should be `handleOpenUrl(_:callbackUrl:isSSO:)` instead of caps `URL`

Reply:
Is it right? master branch uses caps URL now.

Reply:
My bad, I guess I forgot to ask for it to be capitalized instead of all-caps. Thats alright then.